hack/generate-lib-resources.py

```
@@ -150,19 +151,88 @@ def generate_resourcebuilder(directory, types, clients, modifiers, health_checks
-
+        var_name = prop_name[0].lower() + prop_name[1:]
+        lines.extend([
+            '\t{}, err := {}.NewForConfig({})'.format(var_name, data['client_short_name'], new_client_arg),
+            '\tif err != nil {',
+            '\t\treturn nil, err',
+            '\t}',
+        ])
+    lines.extend([
+        '\treturn &clientSet{',
+    ])
+    for prop_name, data in sorted(client_properties.items()):
```

Ruff (0.16.0)

[warning] Loop control variable `data` not used within loop body

(B007)